config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] Great job working on this milestone. It is not professional to keep lines of commented codes in our codebase. kindly delete this commented line of code.